### PR TITLE
Change: Use another shell command to get the latest tag

### PR DESCRIPTION
**Pull request by y0urself**
**What**:

* After reading the [conversation](https://gist.github.com/rponte/fdc0724dd984088606b0) I want to try to use [this](https://gist.github.com/rponte/fdc0724dd984088606b0?permalink_comment_id=3019886#gistcomment-3019886) instead of `git describe --abbrev=0 --tags`.

**Why**:

* This thing with the conventional commits still fails on edge cases and I want it more robust ...

**How**:

**Checklist**:

- [ ] Tests
- [ ] Conventional commit message
- [ ] Documentation

### CODE REVIEW

File: pontos/changelog/conventional_commits.py [hunk 1]
```diff
@@ -95,28 +130,32 @@ def sort_commits(self, commits: List[str]):
...
-            sys.exit(1)
+                        info(f'{commit[0]}: {cleaned_msg}')
         if not commit_dict:
             warning("No conventional commits found.")
             sys.exit(1)
         return commit_dict
 
-    def build_changelog_file(self, commit_dict: Dict):
+    def _build_changelog_file(self, commit_dict: Dict) -> Union[str, None]:
+        """Building the changellog file with the passed dict.
```
Comment (bjoernricks) [inline pontos/changelog/conventional_commits.py:151]:
```suggestion
        """Building the changelog file with the passed dict.
```

**Review by bjoernricks — APPROVED** [review]
